category.html

```diff
@@ -0,0 +1,499 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Explore Top UI Kits Design Assets" />
+    <title>UI Kits Design Assets</title>
+  </head>
+  <body>
+    <!-- Header -->
+    <header>
+      <!-- Logo -->
+      <a href="./index.html" class="logo"> Pixmart.</a>
+
+      <!-- Navigation menu -->
+      <nav>
+        <ul class="nav-menu">
+          <li>
+            <a href="./about.html">About</a>
+          </li>
+          <!-- Dropdown menu of products -->
+          <li>
+            <a href="#" class="dropdown active">Products</a>
+            <ul class="dropdown-content">
+              <li><a href="./category.html">Website</a></li>
+              <li><a href="./category.html">Mobile App</a></li>
+              <li><a href="./category.html">Social Media Feeds</a></li>
+              <li><a href="./category.html">Presentation</a></li>
+              <li><a href="./category.html">Product Mockups</a></li>
+              <li><a href="./category.html">Illustrations</a></li>
+            </ul>
+          </li>
+          <li>
+            <a href="./creator.html">Creators</a>
+            <p class="label">NEW</p>
+          </li>
+          <li><a href="./blog.html">Blog</a></li>
+          <li><a href="./contact.html">Contact</a></li>
+        </ul>
+      </nav>
+
+      <!-- Search bar -->
+      <div class="search-bar">
+        <input type="text" placeholder="Search for Digital Assets" />
+        <button>
+          <img src="/" alt="Search icon" />
+          Search
+        </button>
+      </div>
+
+      <!-- Login and cart buttons -->
+      <div class="buttons">
+        <!-- <button class="btn-account">
```

Comment:
#fixed

Reply:
I have removed login button and left search form. I also asked Liza about footer. She said we can choose one style as well. Since I have chosen homepage 2 design, I apply homepage2 footer design to other pages as well.